packages/y/yosys/package.yml

```diff
@@ -1,9 +1,10 @@
 name       : yosys
-version    : '0.23'
-release    : 18
+version    : '0.35'
```

Comment:
This yosys update seems unrelated to the `klayout` update? Did you mean to include it or was it included accidentally? Neither package seems to have a run or builddep on the other, and while I'm not familiar enough with what they're doing they seem to no be needed for each other? If this was included accidentally, could you please move the yosys update to a separate PR?